deploy/production/ingress.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: justice-gov-uk-production-ingress
+  namespace: justice-gov-uk-production
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: justice-gov-uk-production-ingress-justice-gov-uk-production-green
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: basic-auth-secret
+    nginx.ingress.kubernetes.io/auth-realm: 'Production User | Authentication Required'
+    nginx.ingress.kubernetes.io/proxy-body-size: "200m"
+    nginx.ingress.kubernetes.io/server-snippet: |
+      location = /health {
+        auth_basic off;
+        access_log off;
+        default_type text/plain;
+        add_header Content-Type text/plain;
+        return 200 "OK";
+      }
+      location = /.well-known/security.txt {
+        auth_basic off;
+        return 301 https://raw.githubusercontent.com/ministryofjustice/security-guidance/main/contact/vulnerability-disclosure-security.txt;
+      }
+spec:
+  ingressClassName: default
+  tls:
+  - hosts:
+    - production-justice-gov-uk.apps.live.cloud-platform.service.justice.gov.uk
+  # - hosts:
```

Comment:
We can update the local host file to reach this, is that what we discussed?

Reply:
It was just a thought...

having `production-justice-gov-uk.apps.live.cloud-platform.service.justice.gov.uk` is one way to get the content up for QA.

Then prior to switchover, we change WP_HOME etc. and do a find/replace in the database.